src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php

```diff
@@ -33,13 +36,4 @@
      * @api
      */
     public function collect(Request $request, Response $response, \Exception $exception = null);
```

Comment:
Why you do not move this method definition to the new `DataCollectorInterface` and only keep this interface empty and extending the new one?